handsontable/src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -420,25 +413,44 @@ export class AutoColumnSize extends BasePlugin {
       }
 
       if (!this.hot._getColWidthFromSettings(physicalColumn)) {
-        const samples = this.samplesGenerator.generateColumnSamples(visualColumn, rowsRange);
-
-        samples.forEach((sample, column) => this.ghostTable.addColumn(column, sample));
+        this.addColumnToGhostTableForWidthCalculation(visualColumn, rowsRange);
```

Comment:
I'd be for shorter version and with `#`.
```suggestion
        this.#fillGhostTableWithSamples(visualColumn, rowsRange);
```

Reply:
Shorter name is ok, but if I make it private with `#` I cannot spy on it in test: https://github.com/handsontable/handsontable/pull/11474/files?diff=unified&w=1#diff-9d5698a4d7124ac8c73011a89b9064a9b88df353ca291e3eb963e19ce2f3f5ecR1427

Reply:
I saw that .. that's why I'd propose a different approach to test that. Checking that locally :)

Reply:
Ok, I changed the test according to your idea and renamed the private method